commit 6a290b2895b77be839fcb7c44a6a9879560097ad
Author: Arjun Shankar <arjun@redhat.com>
Date:   Fri Oct 25 09:33:45 2024 +0200

    libio: Correctly link tst-popen-fork against libpthread
    
    tst-popen-fork failed to build for Hurd due to not being linked with
    libpthread.  This commit fixes that.
    
    Tested with build-many-glibcs.py for i686-gnu.
    
    Reviewed-by: Florian Weimer <fweimer@redhat.com>

diff --git a/libio/Makefile b/libio/Makefile
index 38fc72cb8eff193c..570fa1a02b3565c8 100644
--- a/libio/Makefile
+++ b/libio/Makefile
@@ -125,6 +125,8 @@ tests = \
   tst_wscanf \
   # tests
 
+$(objpfx)tst-popen-fork: $(shared-thread-library)
+
 tests-internal = tst-vtables tst-vtables-interposed
 
 ifeq (yes,$(build-shared))
